config/hand.xacro

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<robot xmlns:xacro="http://www.ros.org/wiki/xacro" name="panda">
+    <xacro:macro name="hand">
+        <!--GROUPS: Representation of a set of joints and links. This can be useful for specifying DOF to plan for, defining arms, end effectors, etc-->
+        <!--LINKS: When a link is specified, the parent joint of that link (if it exists) is automatically included-->
+        <!--JOINTS: When a joint is specified, the child link of that joint (which will always exist) is automatically included-->
+        <!--CHAINS: When a chain is specified, all the links along the chain (including endpoints) are included in the group. Additionally, all the joints that are parents to included links are also included. This means that joints along the chain and the parent joint of the base link are included in the group-->
+        <!--SUBGROUPS: Groups can also be formed by referencing to already defined group names-->
+        <group name="hand">
+            <link name="panda_hand"/>
+            <link name="panda_leftfinger"/>
+            <link name="panda_rightfinger"/>
+        </group>
+        <!--GROUP STATES: Purpose: Define a named state for a particular group, in terms of joint values. This is useful to define states like 'folded arms'-->
+        <group_state name="open" group="hand">
+            <joint name="panda_finger_joint1" value="0.04"/>
+        </group_state>
+        <group_state name="close" group="hand">
+            <joint name="panda_finger_joint1" value="0"/>
+        </group_state>
+        <!--END EFFECTOR: Purpose: Represent information about an end effector.-->
+        <!--VIRTUAL JOINT: Purpose: this element defines a virtual joint between a robot link and an external frame of reference (considered fixed with respect to the robot)-->
+        <!--PASSIVE JOINT: Purpose: this element is used to mark joints that are not actuated-->
+        <passive_joint name="panda_finger_joint2"/>
+        <!--DISABLE COLLISIONS: By default it is assumed that any link of the robot could potentially come into collision with any other link in the robot. This tag disables collision checking between a specified pair of links. -->
+        <disable_collisions link1="panda_hand" link2="panda_leftfinger" reason="Adjacent"/>
+        <disable_collisions link1="panda_hand" link2="panda_link4" reason="Never"/>
+        <disable_collisions link1="panda_hand" link2="panda_link5" reason="Default"/>
+        <disable_collisions link1="panda_hand" link2="panda_link6" reason="Never"/>
+        <disable_collisions link1="panda_hand" link2="panda_link7" reason="Default"/>
+        <disable_collisions link1="panda_hand" link2="panda_link8" reason="Adjacent"/>
+        <disable_collisions link1="panda_hand" link2="panda_rightfinger" reason="Adjacent"/>
+        <disable_collisions link1="panda_leftfinger" link2="panda_link4" reason="Never"/>
```

Comment:
I'm not 100% sure that these `disable_collisions` statements are correct. These however are the ones that are given by the `moveit_setup_assistant` when the fingers contain collision geometries.